src/app/views/components/table/table.component.ts

```diff
@@ -42,10 +42,10 @@ const players: IPlayer[] = [
   styleUrls: ['./table.component.scss']
 })
 export class TableComponent implements OnInit, AfterViewInit {
-  @ViewChild('selectAllTemplate')
```

Comment:
This is definitely the main significant breaking change I've encountered, aside from typing issues.  It's subtle, too, because it only errors at runtime.  Is there a way to add explicit notes on something like this to the changelog?  I'll also note it in a separate PR to update the Forge documentation on usage with Angular.

Reply:
It's probably easiest if we edit the release notes and changelog after it's published if we want it to be more explicit.